### PR TITLE
docs(identity): introduce Identity specific documentation relating to multi tenancy

### PR DESCRIPTION
**Pull request by Ben-Sheppard**
## Description

Closes https://github.com/camunda-cloud/identity/issues/2151

This PR introduces the Identity specific docs relating to multi tenancy, specifically:
1. The feature flag
2. The steps of creating and assigning members

## When should this change go live?

- [x] This change is not yet live and should not be merged until {ADD_DATE} (apply `hold` label or convert to draft PR)?
- [ ] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [ ] This functionality is already available but undocumented.
- [ ] This is a bug fix or security concern.

## PR Checklist

- [x] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [x] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

File: docs/self-managed/identity/user-guide/tenants/managing-tenants.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
+---
+id: managing-tenants
+title: "Managing tenants"
+sidebar_label: "Managing tenants"
+description: "Learn about the methods to manage tenants within the Identity application."
```
Comment (christinaausley) [inline docs/self-managed/identity/user-guide/tenants/managing-tenants.md:5]: This is good -- but I would probably say something about _why_ I should manage tenants, for example: "Manage tenants within Identity to XYZ" -- also, we want to refer to Identity as a component 👍

File: docs/self-managed/identity/user-guide/tenants/managing-tenants.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+In this guide you will learn about how to manage tenants in Identity and how to control the members who have access to them.
+
+### Managing tenants
+
+:::note
+The `<default>` tenant is automatically created during Identity startup. See here for more information.
```
Comment (christinaausley) [inline docs/self-managed/identity/user-guide/tenants/managing-tenants.md:16]: Did you mean to provide a link for **here**?

Comment (Ben-Sheppard) [reply docs/self-managed/identity/user-guide/tenants/managing-tenants.md:16]: Ah yes, I meant to actually add a comment here about this, I don't believe the related docs are in place yet (the conceptual side/migration side) so I can't link to anything just yet, should I leave this out?

**Review by christinaausley — COMMENTED** [review]
Technical review in latest commit and two comments to resolve -- nice work!

File: docs/self-managed/identity/user-guide/tenants/managing-tenants.md (hunk 2 above)
Comment (Ben-Sheppard) [reply docs/self-managed/identity/user-guide/tenants/managing-tenants.md:16]: done!

**Review by christinaausley — APPROVED** [review]
Quick work!

**Pull request merged**